teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, _name = 'Unknown')
+    super(4, true)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true if parent_permission == true || parent_permission == false
+  end
```

Comment:
- [ ] I think here this function it is just always return true without any conditions so kindly fix it as required

![image](https://user-images.githubusercontent.com/58553711/193723827-484609f1-5084-42bb-82c9-1fb64b20eba7.png)